streamalert_cli/terraform/rules_engine.py

```diff
@@ -52,7 +52,7 @@ def generate_rules_engine(config):
         'STREAMALERT_PREFIX': prefix,
     }
 
-    if 'log_rule_statistics' in config['lambda']['rules_engine_config']:
+    if config['lambda']['rules_engine_config'].get('log_rule_statistics'):
```

Comment:
Probably we can implement validation against `log_rule_statistics` parameter in the `rules_engine_config` and only allow boolean values. But for now, LGTM.

Reply:
we don't do any validation of these lambda settings now. I'm not about to write a validator just for this :P